includes/admin/home/notices/class-sensei-home-notices.php

```diff
@@ -23,6 +23,13 @@ class Sensei_Home_Notices {
 	 */
 	private $remote_data_api;
 
+	/**
+	 * Screen ID to show notices on.
+	 *
+	 * @var array
```

Comment:
Isn't this a string?

```suggestion
	 * @var string
```